sagemaker-training-compiler/huggingface/tensorflow_multiple_gpu_single_node/language-modeling.ipynb

```diff
@@ -2,7 +2,7 @@
  "cells": [
```

Comment:
<p>Would you help explain more about this memory footprint increase? In our docs, we explain that SMTC should typically help reduce the model memory footprint. However, in the tested models that we've been adding to the documentation, this is not always true. I think we have to explain when SMTC increases the memory footprint but is helpful for faster or more optimized training even with the reduced batch size.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>